include/onnxruntime/core/session/environment.h

```diff
@@ -58,31 +58,31 @@ class Environment {
   /**
    * Registers an allocator for sharing between multiple sessions.
    * Return an error if an allocator with the same OrtMemoryInfo is already registered.
-  */
+   */
   Status RegisterAllocator(AllocatorPtr allocator);
 
   /**
    * Creates and registers an allocator for sharing between multiple sessions.
    * Return an error if an allocator with the same OrtMemoryInfo is already registered.
-  */
+   */
   Status CreateAndRegisterAllocator(const OrtMemoryInfo& mem_info, const OrtArenaCfg* arena_cfg = nullptr);
 
   /**
    * Returns the list of registered allocators in this env.
-  */
+   */
   const std::vector<AllocatorPtr>& GetRegisteredSharedAllocators() const {
     return shared_allocators_;
   }
 
   /**
    * Removes registered allocator that was previously registered for sharing between multiple sessions.
-  */
+   */
   Status UnregisterAllocator(const OrtMemoryInfo& mem_info);
 
+  Environment() = default;
```

Comment:
Why is this change needed ?

Reply:
To use make_unique to replace raw new/delete operator. make_unique can not access private constructors.